### PR TITLE
Incremental rollout test

### PR DESCRIPTION
**Pull request by aitorvs**
Task/Issue URL: https://app.asana.com/0/1203137811378537/1205266828096214/f

### Description
See [asana task](https://app.asana.com/0/1203137811378537/1205266828096214/f)

### Steps to test this PR
see [asana task](https://app.asana.com/0/0/1205278669759816/f)

### CODE REVIEW

**Comment by aitorvs** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #3457** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3457" target="_blank">Graphite</a> 
    * **PR #3462** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3462" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/3462?utm_source=stack-comment).

File: network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/pixels/NetworkProtectionPixelNames.kt [hunk 1]
```diff
@@ -87,4 +87,5 @@ enum class NetworkProtectionPixelNames(
     NETP_WAITLIST_NOTIFICATION_LAUNCHED("m_netp_ev_waitlist_notification_launched_c"),
     NETP_WAITLIST_BETA_ENABLED_DAILY("m_netp_ev_waitlist_enabled_d"),
     NETP_WAITLIST_BETA_ENABLED("m_netp_ev_waitlist_enabled_c"),
+    NETP_INCREMENTAL_ROLLOUT_TEST_PIXEL_DAILY("m_netp_ev_incremental_rollout_test_d"),
```
Comment (karlenDimla) [inline network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/pixels/NetworkProtectionPixelNames.kt:90]: We need to add this in `NetpCohortPixelInterceptor`

File: network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/pixels/NetworkProtectionPixels.kt [hunk 2]
```diff
@@ -377,6 +385,10 @@ class RealNetworkProtectionPixel @Inject constructor(
         firePixel(NETP_WAITLIST_BETA_ENABLED)
     }
 
+    override fun waitlistIncrementalRolloutTest() {
+        tryToFireDailyPixel(NETP_INCREMENTAL_ROLLOUT_TEST_PIXEL_DAILY)
```
Comment (karlenDimla) [inline network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/pixels/NetworkProtectionPixels.kt:389]: I am slightly confused about this. Isn't the spec that we send unique pixel sent once ever? How come we are send a daily pixel?

Comment (karlenDimla) [reply network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/pixels/NetworkProtectionPixels.kt:389]: NIT: Pixel contains appVersion

File: network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/pixels/NetworkProtectionPixelNames.kt (hunk 1 above)
Comment (aitorvs) [reply network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/pixels/NetworkProtectionPixelNames.kt:90]: will do

**Review by karlenDimla — APPROVED** [review]

**Pull request merged**